src/app/learn/learn.component.ts

```diff
@@ -6,7 +6,10 @@ import { Component } from '@angular/core';
 })
 export class LearnComponent {
   public sidebarRoutes: any[] = [
-    { name: 'Overview', path: '/learn' },
+    { name: 'Overview', path: '/learn', children: [
+      { name: 'Test1', path: '/learn/test-1' },
+      { name: 'Test2', path: '/learn/test-2' }
+    ]},
```

Comment:
That's what I was thinking. We can grab the raw routes object and return it in the form that the consuming components expect.